packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1755,14 +1756,15 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @mustCallSuper
   void handleThumbPressStart(Offset localPosition) {
     assert(_debugCheckHasValidScrollPosition());
-    _currentController = widget.controller ?? PrimaryScrollController.of(context);
+    _cachedController = _effectiveScrollController;
```

Comment:
Love this, very nice.

Reply:
@Piinks Hi, this is ready for review now and is not urgent, just put it in your queue.